Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Use an official Python runtime as a parent image
+FROM python:3.9-slim-buster
+
+# Copy the files into the Docker image
+COPY . .
+
+# Install dependencies
+RUN pip install --no-cache-dir fastapi uvicorn[standard] gunicorn qrcode[pil]
```

Comment:
@Mbaoma it would be best to use the requirements.txt here

Reply:
fixed in a new PR